src/main/webapp/app/shared/common/recommended-links/recommended-links.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'jhi-recommended-links',
```

Comment:
Could you rename it to something like `recommended-links-block` or just `recommended-links` instead of using prefix `jhi` because this prefix means it's a built-in component of JHipster which is not correct in your case.